### PR TITLE
🌐 Add Japanese translation for `docs/ja/docs/tutorial/dependencies/sub-dependencies.md`

### PR DESCRIPTION
**Pull request by SwftAlpc**
This PR translates the tutorial/dependencies/sub-dependencies.md to Japanese.
Please review tutorial/dependencies/sub-dependencies.md.

~Issue #1571 generates a new directory to support Japanese.
PR #1656 generates a new directory for tutorial.~
PR #1957 generates a new directoru for dependencies in tutorial.
I will fix dependency after ~#1571, #1656 and~ #1957 are merged.

### CODE REVIEW

File: docs/ja/docs/tutorial/dependencies/sub-dependencies.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+{!../../../docs_src/dependencies/tutorial005.py!}
+```
+
+これはオプショナルのクエリパラメータ`q`を`str`として宣言し、それを返すだけです。
+
+これは非常にシンプルです（あまり便利ではありません）が、サブ依存関係がどのように機能するかに焦点を当てるのに役立ちます。
+
+### 第二の依存関係 「依存可能なもの」と「依存」
+
+そして、別の依存関数（「依存可能なもの」）を作成して、同時にそれ自身の依存関係を宣言することができます（つまりそれ自身も「依存」です）:
```
Comment (tokusumi) [inline docs/ja/docs/tutorial/dependencies/sub-dependencies.md:23]: そして、別の依存関数（「依存可能なもの」）を作成して、同時にそれ自身の依存関係を宣言することができます（つまりそれ自身も「依存」です）:

File: docs/ja/docs/tutorial/dependencies/sub-dependencies.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+query_or_cookie_extractor(["query_or_cookie_extractor"])
+
+read_query["/items/"]
+
+query_extractor --> query_or_cookie_extractor --> read_query
+```
+
+## 同じ依存関係の複数回の使用
+
+依存関係の１つが同じ*path operation*に対して複数回宣言されている場合、例えば、複数の依存関係が共通のサブ依存関係を持っている場合、**FastAPI** はリクエストごとに１回だけそのサブ依存関係を呼び出すことを知っています。
```
Comment (tokusumi) [inline docs/ja/docs/tutorial/dependencies/sub-dependencies.md:62]: 依存関係の１つが同じ*path operation*に対して複数回宣言されている場合、例えば、複数の依存関係が共通のサブ依存関係を持っている場合、**FastAPI** はリクエストごとに１回だけそのサブ依存関係を呼び出します。

**Review by tokusumi — CHANGES_REQUESTED** [review]

**Comment by SwftAlpc** [comment]
@tokusumi 
Thanks for your reveiw!!
Please check it out.

**Review by tokusumi — APPROVED** [review]
Thank you for your work!

**Comment by tiangolo** [comment]
📝 Docs preview for commit 734d90227f0382a0bc67ee541889a41d8badf423 at: https://6499c4ed4a9646007dd7a740--fastapi.netlify.app

**Comment by tiangolo** [comment]
As this was made by @SwftAlpc and approved by @tokusumi, champions of the Japanese translations, I'm gonna merge this right away. 🚀  Thank you! ☕

**Pull request merged**